src/EditorFeatures/VisualBasicTest/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.vb

```diff
@@ -106,6 +106,68 @@ Partial Class c1(Of V As {New}, U)
 End Class")
         End Function
 
+        <WorkItem(17643, "https://github.com/dotnet/roslyn/issues/17643")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)>
+        Public Async Function TestWithDialogNoBackingFields() As Task
+            Await TestWithPickMembersDialogAsync(
+"
+Class Program
+    Public ReadOnly Property F() As Integer
+    [||]
+End Class",
+"
+Class Program
+    Public ReadOnly Property F() As Integer
+
+    Public Overrides Function Equals(obj As Object) As Boolean
+        Dim program = TryCast(obj, Program)
+        Return program IsNot Nothing AndAlso
+               F = program.F
+    End Function
+End Class",
+chosenSymbols:=Nothing)
+        End Function
+
+        <WorkItem(25690, "https://github.com/dotnet/roslyn/issues/25690")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)>
+        Public Async Function TestWithDialogNoIndexer() As Task
```

Comment:
note: this is not an indexer for VB, it's a parameterized property.  To make an indexer, you need to use "default property".  I would consider adding a test for both cases.

Reply:
ok

Reply:
I think i've seen this referred to as an "indexer property" in the docs

Reply:
Internally, we've always called them "parameterized properties". Indexers use the "Default" keyword.